companyweb_base/models/companyweb_credential_wizard.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021-2023 ACSONE SA/NV
+from odoo import fields, models
+from odoo.exceptions import ValidationError
+
+
+class CompanyWebCredentialWizard(models.TransientModel):
+
+    _name = "companyweb_base.credential_wizard"
+    _description = "Ask for CompanyWeb login & password"
+    cweb_login = fields.Char("Company Web Login")
+    cweb_password = fields.Char("Company Web Password")
+
+    def get_credentials(self):
```

Comment:
This methode is not "getting credentials" so another name would be better.

Reply:
method's name changed to save_cweb_login_pwd
I also changed the view, now the button is labelled "Save" in place of "Register"